src/main/java/com/softserve/config/DBConfig.java

```diff
@@ -103,4 +106,13 @@ public HibernateTransactionManager getTransactionManager() {
         return transactionManager;
     }
 
+    @Bean
+    public SpringLiquibase liquibase() {
+        SpringLiquibase liquibase = new SpringLiquibase();
+        liquibase.setChangeLog("classpath:db/changelog/db.changelog-master.yaml");
+        liquibase.setDataSource(getDataSource());
+        liquibase.setShouldRun(Objects.requireNonNull(environment.getProperty("spring.liquibase.enabled", Boolean.class)));
```

Comment:
You wouldn't like to receive a NPE here, hence, you might need to avoid using the requiredNonNull method and use the overloaded getProperty method with default value (Boolean.TRUE in this case)